scout/server/blueprints/variants/views.py

```diff
@@ -256,13 +257,17 @@ def sv_variants(institute_id, case_name):
 
     # Setup variant count session with variant count by category
     controllers.variant_count_session(store, institute_id, case_obj["_id"], variant_type, category)
-    session["filtered_variants"] = variants_query.count()
+    query = store.get_query(case_obj["_id"], form.data, None, category)
+    result_size = variants_query.collection.count_documents(query)
```

Comment:
Why don't you use the store.variants_count that you created in scout/adapter/mongo/variant.py to do that instead?